pages/HomePage.js

```diff
@@ -0,0 +1,22 @@
+import BasePage from '../pages/BasePage';
+
+class HomePage extends BasePage {
+
+  //WebElements
+  get title(){ return $('#page-title') }
+
+  get userConf() { return $('img[alt="User Avatar"]')};
+
+  get logOut() {return $('a[href="/bank/logout"]')}
+
+
+  async settings() {
```

Comment:
In simlar spirit to previous comment, this is a method, you are modeling an **operation** here, an **action** from the object, therefore the name should represent the action implied here: In this case, something like: **accessUserSettings()** which is more descriptive, and, when seen from the client code (the test that calls to this action), it is inmediately obvious what is happening, removing the need to go check the method definition to see whats going on (this efficiency, when codebase grows, saves you hours, if not days of development time, as you spend most of your programming time **reading code**, instead of writing it)